pkg/controller/rest/vdri/operation.go

```diff
@@ -81,6 +91,49 @@ func (o *Operation) CreatePublicDID(rw http.ResponseWriter, req *http.Request) {
 	rest.Execute(o.command.CreatePublicDID, rw, bytes.NewReader(reqBytes))
 }
 
+// SaveDID swagger:route POST /vdri/did vdri saveDIDReq
+//
+// Saves a did document with the friendly name.
+//
+// Responses:
+//    default: genericError
+//        200: emptyRes
+func (o *Operation) SaveDID(rw http.ResponseWriter, req *http.Request) {
+	rest.Execute(o.command.SaveDID, rw, req.Body)
+}
+
+// GetDID swagger:route GET /vdri/did/{id} vdri getDIDReq
+//
+// Gets did document with the friendly name.
+//
+// Responses:
+//    default: genericError
+//        200: documentRes
+func (o *Operation) GetDID(rw http.ResponseWriter, req *http.Request) {
+	id := mux.Vars(req)["id"]
+
+	decodedID, err := base64.StdEncoding.DecodeString(id)
+	if err != nil {
+		rest.SendHTTPStatusError(rw, http.StatusBadRequest, vdri.InvalidRequestErrorCode, err)
```

Comment:
may be wrap error with some message? just to let caller know where it went wrong, something like "invalid id"

Reply:
done